src/GraphQLController.php

```diff
@@ -86,6 +98,43 @@ protected function queryContext(string $query, ?array $params, string $schema)
         }
     }
 
+    /**
+     * @param  string  $schema
+     * @param  array  $input
+     * @return string
+     * @throws Error\AutomaticPersistedQueriesError
+     * @throws \Psr\SimpleCache\InvalidArgumentException
+     */
+    protected function getQueryFromAPQIfNeeded(string $schema, array $input): string
```

Comment:
About `$schema` I thought it would be useful to use it as a namespace for the cache key (I had probably forgotten that)
```
$apqCacheIdentifier = implode('.', [$apqCachePrefix, $schema, $hash]);
```
It's ok for you?

Reply:
I like the idea! Make sense in case someone inspects the cache keys and has multiple schemas, there's a sense of where it belongs to 👍 

PS: could you rename it to `$schemaName` though?

It's not this PRs fault, I think existing uses of "schema" vs. "schema name" could be clarified better too, will take care of this another time.

Reply:
Ok